### PR TITLE
docs plcontainer - further edits

### PR DESCRIPTION
**Pull request by lenapivotal**
Continued with edits from google doc reviews and PR 
https://github.com/greenplum-db/gpdb/pull/9768

Refreshed staging app:
http://docs-lena-plcontainer.cfapps.io/7-0/analytics/pl_container.html

### CODE REVIEW

File: gpdb-doc/dita/analytics/analytics.ditamap [hunk 1]
```diff
@@ -13,12 +13,11 @@
...
     <topicref href="intro.xml" navtitle="Procedural Languages" linking="none">
     <topicref href="pl_container.xml" navtitle="PL/Container" linking="none">
-          <topicref href="pl_container.xml#about_pl_container" navtitle="About PL/Container Language Extension" linking="none"/>
-          <topicref href="pl_container.xml#topic3" navtitle="Install PL/Container" linking="none"/>
-          <topicref href="pl_container.xml#upgrade_plcontainer" navtitle="Upgrade PL/Container" linking="none"/>
-          <topicref href="pl_container.xml#uninstall_plcontainer" navtitle="Unistall PL/Container" linking="none"/>
-          <topicref href="pl_container.xml#topic_kds_plk_rbb" navtitle="PL/Container References" linking="none"/>
-          <topicref href="pl_container_using.xml" linking="none"/>
+          <topicref href="pl_container.xml#about_pl_container" navtitle="About the PL/Container Language Extension" linking="none"/>
+          <topicref href="pl_container.xml#topic3" navtitle="Uninstalling PL/Container" linking="none"/>
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/analytics.ditamap]: Looks like you're changing this from Install to Uninstalling.  I'm not sure that was the intention.  For all of these, though, if you leave out the navtitle then the title will be automatically pulled in.

File: gpdb-doc/dita/analytics/analytics.ditamap [hunk 2]
```diff
@@ -13,12 +13,11 @@
...
+          <topicref href="pl_container.xml#upgrade_plcontainer" navtitle="Upgrading PL/Container" linking="none"/>
+          <topicref href="pl_container.xml#uninstall_plcontainer" navtitle="Unistalling PL/Container" linking="none"/>
```
Comment (dyozie) [inline gpdb-doc/dita/analytics/analytics.ditamap]: This is a second Uninstalling title (also typo here - Unistalling).

File: gpdb-doc/dita/analytics/analytics.ditamap (hunk 1 above)
Comment (lenapivotal) [reply gpdb-doc/dita/analytics/analytics.ditamap]: I just added a "the", not sure what else changed... I also added the "using...."

**Comment by dyozie** [comment]
Thanks Lena - that looks better!

**Pull request merged**